index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Microverse-Student-Project-1</title>
+  </head>
+  <body>
+    <header>
+      <div class="nav-bar">
+        <div class="for-logo">
+          <a class="logo" href="#">Tom Daniel</a>
+        </div>
```

Comment:
- [ ] [OPTIONAL] I noticed that your navbar logo and menu-hamburger button is not well alligned on the cross-axis. I realize that it is caused by a few CSS declarations that you have given to the `a` tag. After I removed them, it became better as you see in the second image below. 

```css
.for-logo a {
    /* width: 76px; Remove */
    /* height: 20px; Remove */
    font-family: "Poppins", sans-serif;
    font-style: normal;
    font-weight: 700;
    font-size: 18px;
    line-height: 20px;
    color: #6070ff;
    text-decoration: none;
}

.logo {
    display: flex;
    justify-content: flex-start;
    font-family: 'Poppins', sans-serif;
    font-style: normal;
    font-weight: 700;
    font-size: 18px;
    line-height: 20px;
    color: #6070ff;
    text-decoration: none;
    /* margin-top: 15px; Remove */
}
```
Your design currently
![image](https://user-images.githubusercontent.com/84156714/219381636-87263954-12dc-4255-8e75-11593ac2bd1b.png)

After I remove the styles
![image](https://user-images.githubusercontent.com/84156714/219382761-3130e440-f059-4323-8315-c73c0e8dafba.png)